learning_wavelets/tests/models/exact_recon_unet_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import tensorflow as tf
+import tensorflow.keras.backend as K
+from tensorflow.keras.optimizers import Adam
+
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+def test_init():
+    ExactReconUnet()
+    K.clear_session()
+
+def test_fit():
+    model=ExactReconUnet(n_output_channels=1, kernel_size=3, layers_n_channels=[4, 8, 16, 32])
+    model.compile(optimizer=Adam(lr=1e-3), loss='mse')
+    model.fit(
+        x=(tf.random.normal((8, 32, 32, 1), seed=0),
+           tf.random.normal((8, 1, 1, 1), seed=0)),
```

Comment:
I think it's better to handle multi-line statements in the following way
```suggestion
        x=(
        	tf.random.normal((8, 32, 32, 1), seed=0),
            tf.random.normal((8, 1, 1, 1), seed=0),
        ),
```

It does use more lines, but you can always fold them in your editor, but more importantly, it limits the future changes in case one wants to change the name of the argument `x` to something longer.